index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
I was very confused by the same thing.

> An "aud" (Audience) claim in the token MUST include the unicode
> serialization of the origin (Section 6.1 of [RFC6454]) of the push
> resource URL.  This binds the token to a specific push service.

Everyone I've spoken to regarding this matter thought it was the origin of the application server and after reading the spec wasnt sure what it should or could be. It's definitely one of those moments where the spec is phrased awkwardly, but FCM will reject and the final sentence in the spec regarding audience does suggest it should be the push service